doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -288,6 +288,8 @@ Serial LTE modem
 * Added an overlay :file:`overlay-memfault.conf` file to enable Memfault.
   For more information about Memfault features in |NCS|, see :ref:`mod_memfault`.
 * Updated the application to use the :ref:`lib_downloader` library instead of the deprecated :ref:`lib_download_client` library.
+* Updated in Zephyr the numerical values of various |NCS|-specific socket options that are used with the ``#XSOCKETOPT`` command.
+  For more information, refer to the updated :ref:`SLM_AT_SOCKET` documentation.
```

Comment:
```suggestion
* Updated:

  * The application to use the :ref:`lib_downloader` library instead of the deprecated :ref:`lib_download_client` library.
  * In Zephyr, the numerical values of various |NCS| specific socket options that are used with the ``#XSOCKETOPT`` command.
    For more information, refer to the updated :ref:`SLM_AT_SOCKET` documentation.
```